sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -506,15 +505,38 @@ def test_any_sequencer(self):
 
     self._test_pipeline_workflow(any_sequence_pipeline, 'any_sequencer.yaml')
 
+  def _test_pipeline_workflow_inlined_spec(self,
+                              pipeline_function,
+                              pipeline_yaml,
+                              normalize_compiler_output_function=None,
+                              tekton_pipeline_conf=TektonPipelineConf()):
+    test_data_dir = os.path.join(os.path.dirname(__file__), 'testdata')
+    golden_yaml_file = os.path.join(test_data_dir, pipeline_yaml.replace(".yaml", "") + "_inlined.yaml")
```

Comment:
So because we are making all the yamls to be inline by default, maybe we only append suffix `_noinlined.yaml` for the non inline yaml. The inline ones can be just `.yaml`.

Reply:
Done.